bftcosi/packets.go

```diff
@@ -70,6 +71,10 @@ func (bs *BFTSignature) Verify(s network.Suite, publics []kyber.Point) error {
 	origCommit := s.Point()
 	pointLen := s.PointLen()
 	sigLen := pointLen + s.ScalarLen()
+
+	if len(bs.Sig) < sigLen {
```

Comment:
Shouldn't this be `!=`?

Reply:
No, there's a variable length mask appended onto the end (cosi.go, line 204).